src/main/docker/Dockerfile

```diff
@@ -15,25 +15,36 @@
 #
 
 ARG DOCKER_HUB_PUBLIC=docker.io
-FROM ${DOCKER_HUB_PUBLIC}/opensuse/leap:latest
-
-# Set the locale manually to a set inherited from the base image (defaults to POSIX)
-ENV LANG=en_US.utf8
+FROM ${DOCKER_HUB_PUBLIC}/opensuse/leap:latest as builder
 
-# Update the OS packages, install cURL, postgreSQL client and dejavu-fonts
+# Update the OS packages with dirmngr and curl to enable gosu install
 RUN zypper -n refresh && \
     zypper -n update && \
-    zypper -n install curl postgresql dejavu-fonts && \
+    zypper -n install dirmngr curl && \
     zypper -n clean --all
 
 # Install gosu
 RUN gpg --batch --keyserver-options http-proxy=${env.HTTP_PROXY} --keyserver hkps://keys.openpgp.org \
       --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 && \
     curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.12/gosu-amd64" && \
     curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.12/gosu-amd64.asc" && \
-    gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu && \
-    rm /usr/local/bin/gosu.asc && \
-    chmod +x /usr/local/bin/gosu
+    gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu
+
+ARG DOCKER_HUB_PUBLIC
+FROM ${DOCKER_HUB_PUBLIC}/opensuse/leap:latest
+
+# Set the locale manually to a set inherited from the base image (defaults to POSIX)
+ENV LANG=en_US.utf8
+
+# Update the OS packages, install postgreSQL client and dejavu-fonts
+RUN zypper -n refresh && \
+    zypper -n update && \
+    zypper -n install postgresql dejavu-fonts && \
```

Comment:
Curl was here before the Gosu install so can you put that back in here so the built image has it?